<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>information_customer</class>
 <widget class="QWidget" name="information_customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>مشخصات</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #34495e;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:25px;	</string>
   </property>
   <property name="text">
    <string>مشخصات: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>140</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>نام:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>200</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>نام خوانوادگی:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>320</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>تاریخ تولد:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>کد ملی:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>390</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>نام کاربری:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>450</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>مرز عبور:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_customer">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>150</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black; font-size:18px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="last_name_customer">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black; font-size:18px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="n_cod_customer">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>270</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black; font-size:18px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="birth_day_customer">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>330</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black; font-size:18px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="password_customer">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>460</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black; font-size:18px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="username_customer">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>400</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black; font-size:18px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:25px;	</string>
   </property>
   <property name="text">
    <string>حساب ها:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>471</width>
     <height>621</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>469</width>
      <height>619</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="accounts">
       <property name="styleSheet">
        <string notr="true">color:black; font-size:18px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="back_menu_pc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d;  font-size:18px;</string>
   </property>
   <property name="text">
    <string>برگشت</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
